experiment/build/test_builder.py

```diff
@@ -62,6 +63,8 @@ def get_benchmarks_or_fuzzers(benchmarks_or_fuzzers_directory, filename,
     ]
 
 
+@pytest.mark.skipif(not sys.version.startswith('3.7.7'),
```

Comment:
Why need 3.7.7 exact, should check major =3 and minor =7. it is easy to go higher version in 3.7.

Reply:
Done.

Reply:
Done.